Keas.Mvc/Views/Report/UnAcceptedItems.cshtml

```diff
@@ -4,14 +4,6 @@
     ViewData["Title"] = "Items Pending Acceptance";
 }
 
-@section Styles {
-    @await Html.PartialAsync("_ReportStylePartial")
-}
-
-@section Scripts {
-   @await Html.PartialAsync("_ReportScriptsPartial")                             
```

Comment:
Instead of custom doing the report scripts, why not just make all of the reports exportable?